tests/integ/scala/test_udf_suite.py

```diff
@@ -549,6 +550,59 @@ def geometry(g):
     )
 
 
+@pytest.mark.xfail(reason="SNOW-974852 vectors are not yet rolled out", strict=False)
```

Comment:
We can instead flip the param on the small set of preprod/prod accounts that run these tests.  ?